app/Repositories/PhotoRepository.php

```diff
@@ -42,10 +42,17 @@ public function __construct(AWS $aws, Registrar $registrar)
      */
     public function create(array $data, $signupId)
     {
+        // Include signup_id when we create the Event
+        $data['signup_id'] = $signupId;
+
         // Don't send quantity and why_participated - we don't want this to live on the post_photo event.
         $postEvent = Event::create(array_except($data, ['quantity', 'why_participated']));
 
-        $fileUrl = $this->aws->storeImage($data['file'], $signupId);
+        if (isset($data['file'])) {
+            $fileUrl = $this->aws->storeImage($data['file'], $signupId);
+        } else {
+            $fileUrl = 'default';
+        }
 
         $editedImage = $this->crop($data, $signupId);
```

Comment:
@chloealee great question! It hasn't caused any crashes for me, but we definitely don't need to try to set `$editedImage` for broken ones, so I'm pulling it up into the `if`. Good catch!